pkg/reconciler/reconciler.go

```diff
@@ -39,6 +38,7 @@ func (r Updater) Reconcile(ctx context.Context, c client.Client, scheme *runtime
 			r.resource.GetNamespace(), r.resource.GetName(),
 			r.resource.GetObjectKind().GroupVersionKind().String(), err)
 	}
+
```

Comment:
Thanks also for this style clean up. I'll try to have that automatically applied for the whole project ;)